apis/submariner/v1alpha1/submariner_types.go

```diff
@@ -87,6 +89,15 @@ type DaemonSetStatus struct {
 	MismatchedContainerImages bool                     `json:"mismatchedContainerImages"`
 }
 
+type HealthCheckSpec struct {
+	HealthCheckEnabled bool `json:"healthCheckEnabled,omitempty"`
+	// The interval upon which HealthChecker ping packets are sent
+	HealthCheckIntervalSeconds uint64 `json:"healthCheckIntervalSeconds,omitempty"`
+	//  Defines how long we should wait for HealthChecker ping packets before we
+	//  declare the connection as down
+	HealthCheckTimeoutSeconds uint64 `json:"healthCheckTimeoutSeconds,omitempty"`
```

Comment:
This should be `MaxLosablePackets`.    

Also I think we can drop the HealthCheck prefix for the fields as its in the containing Spec.

Reply:
Isn't it more automatic with timeoutSeconds and we calculate the number of packets?

otherwise if admin changes the interval it will need to recalculate "MaxLosablePackets" ? 

anyway... opinion... every of them has it's merits (like MaxLosablePackets will be quite exact).